clusterloader/framework/util.go

```diff
@@ -21,19 +21,55 @@ import (
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
 const maxRetries = 5
 
+// CreateNS creates a namespace if it is new, otherwise it will return the existing namespace pointer
```

Comment:
As I expect this file to grow a lot, it's probably worth splitting it in front. Please create pod_utils.go, rc_utils.go and leave only generic utils in 'utils.go'. Hopefully that will be enough.